### PR TITLE
Dolphin update again

### PR DESCRIPTION
**Pull request by osfanbuff63**
Draft of redoing parts of #257, I'll be working on this when I have time. ~~Does not work yet~~.

Edit: (5/27) Should work, but Jekyll has broken on me so many times lol.

### CODE REVIEW

**Comment by osfanbuff63** [comment]
@Peter0x44 Sorry to bother you, but I can't seem to run `bundle exec jekyll serve`, with this error:

<details>
  <summary>Error</summary>
```

PS C:\Users\Username\Desktop\Wii-Guide> bundle exec jekyll serve --trace
Calling `DidYouMean::SPELL_CHECKERS.merge!(error_name => spell_checker)' has been deprecated. Please call `DidYouMean.correct_error(error_name, spell_checker)' instead.
Calling `DidYouMean::SPELL_CHECKERS.merge!(error_name => spell_checker)' has been deprecated. Please call `DidYouMean.correct_error(error_name, spell_checker)' instead.
Configuration file: C:/Users/Username/Desktop/Wii-Guide/_config.yml
  Dependency Error: Yikes! It looks like you don't have jekyll-paginate-v2 or one of its dependencies installed. In order to use Jekyll as currently configured, you'll need to install this gem. If you've run Jekyll with `bundle exec`, ensure that you have included the jekyll-paginate-v2 gem in your Gemfile as well. The full error message from Ruby is: 'cannot load such file -- jekyll-paginate-v2' If you run into trouble, you can find helpful resources at https://jekyllrb.com/help/!
C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/jekyll-4.2.2/lib/jekyll/external.rb:73:in `rescue in block in require_with_graceful_fail': jekyll-paginate-v2 (Jekyll::Errors::MissingDependencyException)
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/jekyll-4.2.2/lib/jekyll/external.rb:58:in `block in require_with_graceful_fail'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/jekyll-4.2.2/lib/jekyll/external.rb:57:in `each'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/jekyll-4.2.2/lib/jekyll/external.rb:57:in `require_with_graceful_fail'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/jekyll-4.2.2/lib/jekyll/plugin_manager.rb:30:in `require_gems'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/jekyll-4.2.2/lib/jekyll/plugin_manager.rb:22:in `conscientious_require'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/jekyll-4.2.2/lib/jekyll/site.rb:131:in `setup'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/jekyll-4.2.2/lib/jekyll/site.rb:36:in `initialize'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/jekyll-4.2.2/lib/jekyll/commands/build.rb:30:in `new'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/jekyll-4.2.2/lib/jekyll/commands/build.rb:30:in `process'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/jekyll-4.2.2/lib/jekyll/command.rb:91:in `block in process_with_graceful_fail'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/jekyll-4.2.2/lib/jekyll/command.rb:91:in `each'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/jekyll-4.2.2/lib/jekyll/command.rb:91:in `process_with_graceful_fail'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/jekyll-4.2.2/lib/jekyll/commands/serve.rb:86:in `block (2 levels) in init_with_program'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `block in execute'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `each'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `execute'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/mercenary-0.4.0/lib/mercenary/program.rb:44:in `go'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/mercenary-0.4.0/lib/mercenary.rb:21:in `program'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/jekyll-4.2.2/exe/jekyll:15:in `<top (required)>'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/bin/jekyll:25:in `load'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/bin/jekyll:25:in `<main>'
C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/jekyll-4.2.2/lib/jekyll/external.rb:60:in `require': cannot load such file -- jekyll-paginate-v2 (LoadError)
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/jekyll-4.2.2/lib/jekyll/external.rb:60:in `block in require_with_graceful_fail'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/jekyll-4.2.2/lib/jekyll/external.rb:57:in `each'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/jekyll-4.2.2/lib/jekyll/external.rb:57:in `require_with_graceful_fail'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/jekyll-4.2.2/lib/jekyll/plugin_manager.rb:30:in `require_gems'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/jekyll-4.2.2/lib/jekyll/plugin_manager.rb:22:in `conscientious_require'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/jekyll-4.2.2/lib/jekyll/site.rb:131:in `setup'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/jekyll-4.2.2/lib/jekyll/site.rb:36:in `initialize'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/jekyll-4.2.2/lib/jekyll/commands/build.rb:30:in `new'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/jekyll-4.2.2/lib/jekyll/commands/build.rb:30:in `process'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/jekyll-4.2.2/lib/jekyll/command.rb:91:in `block in process_with_graceful_fail'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/jekyll-4.2.2/lib/jekyll/command.rb:91:in `each'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/jekyll-4.2.2/lib/jekyll/command.rb:91:in `process_with_graceful_fail'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/jekyll-4.2.2/lib/jekyll/commands/serve.rb:86:in `block (2 levels) in init_with_program'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `block in execute'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `each'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `execute'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/mercenary-0.4.0/lib/mercenary/program.rb:44:in `go'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/mercenary-0.4.0/lib/mercenary.rb:21:in `program'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/gems/jekyll-4.2.2/exe/jekyll:15:in `<top (required)>'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/bin/jekyll:25:in `load'
        from C:/Users/Username/Desktop/Wii-Guide/vendor/bundle/ruby/3.1.0/bin/jekyll:25:in `<main>'

<details>

Trying to set it up so I can test, and not have issues like we did in #257 :P

File: _pages/en_US/riiconnect24-dolphin.md [hunk 1]
```diff
@@ -30,9 +34,38 @@ If you have Dolphin already installed, skip to Section II
 4. Press on `Tools` -> `Perform Online System Update` -> Choose your region
 ![Perform Online System Update](/images/Dolphin_RC24/1.jpg)
 
-##### Section II - Installing RiiConnect24.
+If you have a homebrewed Wii console, you can use a [BootMii NAND dump](bootmii) instead of installing the Wii System menu via this method.
```
Comment (osfanbuff63) [inline _pages/en_US/riiconnect24-dolphin.md:37]: Maybe we should add either a section or a page for how to import a BootMii NAND dump into Dolphin, or link to the [Dolphin Wiki](https://wiki.dolphin-emu.org/index.php?title=NAND_Usage_Guide) page?

File: _pages/en_US/riiconnect24-dolphin.md [hunk 2]
```diff
@@ -41,22 +74,46 @@ If you have Dolphin already installed, skip to Section II
...
+9. After it's done, there will a be a `Mii Contest Channel (Europe) (Channel) (RiiConnect24).wad` or `Check Mii Out Channel (USA) (Channel) (RiiConnect24).wad` file in WAD folder next to RiiConnect24Patcher.bat
+10. In Dolphin, press `Tools` and then `Install WAD`, and select the `Mii Contest Channel (Europe) (Channel) (RiiConnect24).wad` or `Check Mii Out Channel (USA) (Channel) (RiiConnect24).wad`.
 
-You're all done! Unfortunately, Nintendo Channel and Wii Mail don't work in Dolphin yet.
+</div>
+
+You're all done! Unfortunately, the Nintendo Channel and Wii Mail don't work in Dolphin yet.
 {: .notice--info}
+
+<script>
```
Comment (osfanbuff63) [inline _pages/en_US/riiconnect24-dolphin.md:96]: I believe this is what I left out in #257

**Comment by osfanbuff63** [comment]
I think this is ready, merge whenever you get to it lol

**Comment by Peter0x44** [comment]
I should be able to review it within the week. Thanks.

File: _pages/en_US/riiconnect24-dolphin.md [hunk 3]
```diff
@@ -30,9 +34,38 @@ If you have Dolphin already installed, skip to Section II
...
+{: .notice--info}
+8. Press any key to go back to the RiiConnect24 patcher.
+9. Once back in the RiiConnect24 Patcher, press `1` and then `Enter`.
+10. Select your region (Europe or USA). The patcher will now begin to patch the WAD files.
+11. Once it's done, press 2, (this will close the patcher) and then go to the directory that `RiiConnect24Patcher.bat` is in. There should be be a `Mii Contest Channel (Europe) (Channel) (RiiConnect24).wad` or `Check Mii Out Channel (USA) (Channel) (RiiConnect24).wad`, as well as an `Everybody Votes Channel ([your-selected-region]) (Channel) (RiiConnect24).wad` file in the WAD folder there.
+12. In Dolphin, press `Tools` and then `Install WAD`, and select the `Mii Contest Channel (Europe) (Channel) (RiiConnect24).wad` or `Check Mii Out Channel (USA) (Channel) (RiiConnect24).wad`. Do the same for `Everybody Votes Channel ([your-selected-region]) (Channel) (RiiConnect24).wad`.
+</div>
+
+<div id="unix" class="blanktabcontent" markdown="1">
+1. Run `VFF-Downloader-for-Dolphin.sh` which you downloaded [here](https://github.com/RiiConnect24/.VFF-File-Downloader-for-Dolphin/releases)
```
Comment (Peter0x44) [inline _pages/en_US/riiconnect24-dolphin.md:68]: I know this was already there before, but I'm not really a fan of the "which you downloaded here"
not really sure there's much point of putting the link twice.

File: _pages/en_US/riiconnect24-dolphin.md [hunk 4]
```diff
@@ -12,11 +12,15 @@ If you need help for anything regarding this tutorial, please directly contact K
...
-* A computer with either Windows 7 or newer or any Unix based system
-* [.VFF-File-Downloader-for-Dolphin](https://github.com/RiiConnect24/.VFF-File-Downloader-for-Dolphin/releases)
-* [Dolphin](https://dolphin-emu.org/download/) 
+
+* A computer with either Windows 7 or newer or any Unix-based system
+* [.VFF-File-Downloader-for-Dolphin](https://github.com/RiiConnect24/.VFF-File-Downloader-for-Dolphin/releases) if using a Unix-based system
+* [Dolphin](https://dolphin-emu.org/download/)
 * [RiiConnect24 Patcher](https://github.com/RiiConnect24/RiiConnect24-Patcher/releases)
 
+The Unix tutorial will work on Windows, but it is recommended to use the Windows-only method instead if you have the option, since it is generally easier to understand.
```
[the remark this answers is not in the document]
Comment (osfanbuff63) [reply _pages/en_US/riiconnect24-dolphin.md:21]: What's possible is downloading the batch file manually for vff Downloader, but that just isn't needed with all you really need to download is the rc24 patcher with the windows method.

File: _pages/en_US/riiconnect24-dolphin.md (hunk 3 above)
Comment (osfanbuff63) [reply _pages/en_US/riiconnect24-dolphin.md]: 👍🏻

File: _pages/en_US/riiconnect24-dolphin.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (Peter0x44) [reply _pages/en_US/riiconnect24-dolphin.md:21]: OK, I think this is worded confusingly. it's not necessary, you shouldn't be using the "unix tutorial" if you are on windows anyway.

**Comment by Devnol** [comment]
Hey, I saw your Issue at Jekyll's repo and I was wondering if you have any idea as to why it throws that error. I tested it locally and going as far back as commit 5a45608 it works fine, so I assume it has something to do with Jekyll being updated from then to now. I went back to the master current commit and downgraded jekyll to 3.9, also removing jemoji that threw some errors and didn't even seem to be in use anyway and it appears to work now. If you have any other insight that would be great.

**Comment by osfanbuff63** [comment]
Another thing is, do we use a custom version of minimal-mistakes? Even if we do, we can keep the overridden files and make it a Gemfile dependency, but idk. I'll get this PR finished though.

**Comment by Devnol** [comment]
I'm really not sure, I don't think we do. But yeah finish this PR and if you want you can help me with it on the one I'll open later

**Comment by Peter0x44** [comment]
LGTM, I approve

**Pull request merged**